scenario/AddMaticxCollateralScenario.ts

```diff
@@ -0,0 +1,185 @@
+import { scenario } from './context/CometContext';
+import { expect } from 'chai';
+import { utils } from 'ethers';
+import { exp } from '../test/helpers';
+import { COMP_WHALES, calldata } from '../src/deploy';
+import { impersonateAddress } from '../plugins/scenario/utils';
+import { fastL2GovernanceExecute, matchesDeployment } from './utils';
+import { BaseBridgeReceiver } from '../build/types';
+import { World } from '../plugins/scenario';
+
+const MATICX_WHALES = {
+  polygon: ['0x68B9220B8E617b7700aCAE1a5Ff43F3eb29257F3'],
+};
+
+scenario.only(
+  'add new asset maticx',
+  {
+    filter: async (ctx) =>
+      matchesDeployment(ctx, [{ network: 'polygon' }, { network: 'mumbai' }]),
```

Comment:
This won't work for mumbai right? The addresses you used for MATICX and the pricefeed are only for Polygon.